playlists/registry/37i9dQZEVXcPvtTWQjFLSm

```diff
@@ -0,0 +1 @@
+
```

Comment:
A lot of people have their own "Discover Weekly" playlists. Can you add a line to this file so that we can differentiate between them?
```
Discover Weekly (@milomilomilo)
```